ahoopes16/tidkinbrae
# Contact Information

### Summary
This PR is to add my contact info in a way that is clean and actionable.

If accepted, this PR will:
- Add my name
- Add my email address
- Add my phone number
- Add my GitHub profile
- Add my StackOverflow profile
- Change the favicon to a quick and dirty one because the Gatsby one was bothering me

### To Test
1. `yarn && yarn start`
1. Visit http://localhost:8000
1. On the left side of the page, you should see all of the contact information listed above.
1. Click on each of them
   1. Email should open up your default mailer and start composing an email to me
   1. Phone number should prompt you to choose an application to call me
   1. GitHub profile should take you to my GitHub profile
   1. StackOverflow profile should take you to my StackOverflow profile
